rosplan_turtlebot2_demo/config/waypoints.yaml

```diff
@@ -0,0 +1,14 @@
+# waypoints reference frame
+waypoint_frameid: map
+
+# waypoint 2D coordinates: x, y, theta
+wp0: [0, 1, 2.5]
```

Comment:
Wouldn't it be better to have a super key "waypoints" with the wpX inside? That way we can easily iterate over them

Reply:
launch file + yaml file slightly changed to enhance readability